uber/templates/accounts/index.html

```diff
@@ -67,8 +67,8 @@
         <td>
             <form class="form update-form" method="post" action="update">
                 <input type="hidden" name="id" value="{{ account.id }}" />
-                {% csrf_token %}
-                <div class="form-group">{% checkgroup account.access %}</div>
+                {{ csrf_token() }}
+                <div class="form-group">{{ account.html_checkgroup('access') }}</div>
```

Comment:
yea btw, see above comments about why this would have to instead be ```{{ checkgroup(account, 'access') }}```.  Unlike Django, if we do ``{{ checkgroup(account.access) }}```, we only get the the value of 'access', and have no way to get to 'account' to do things with that underlying model.  

This is because Jinja immediately resolves ```account.access``` as a variable, whereas django give you the string ```account.access``` and then you can do stuff to it like parse out 'acccount' and 'access'